x-pack/libbeat/docker-compose.yml

```diff
@@ -40,8 +40,11 @@ services:
       - "xpack.security.enabled=true"
       - "xpack.license.self_generated.type=trial"
     command: bash -c "bin/elasticsearch-keystore create && echo changeme | bin/elasticsearch-keystore add --stdin bootstrap.password && /usr/local/bin/docker-entrypoint.sh eswrapper"
+    restart: on-failure:5
 
   kibana:
+    depends_on:
```

Comment:
This will make startup slower as it only starts KB when ES is ready. Let's see if it helps.